backend/config/config.py

```diff
@@ -101,4 +102,34 @@ def _load_config(path=None):
     'geocoder': 'oxcoder',
     'measurer': 'haversine',
     'number_of_adequacy_processors': 8,
+    'logging': {
+        'version': 1,
+        'disable_existing_loggers': True,
+        'formatters': {
+            'console': {
+                'format': '[%(asctime)s][%(levelname)s] %(name)s '
+                          '%(filename)s:%(funcName)s:%(lineno)d | %(message)s',
+                'datefmt': '%H:%M:%S',
+            },
+        },
+        'handlers': {
+            'console': {
+                'level': 'DEBUG',
+                'class': 'logging.StreamHandler',
+                'formatter': 'console'
+            },
+            'sentry': {
+                'level': 'DEBUG',
+                'class': 'raven.handlers.logging.SentryHandler',
+                'dsn': os.environ.get('SENTRY_DSN', None),
```

Comment:
have you tried using a piece of config for another piece in C2Q?  we use SENTRY_DSN in multiple places. Currently just directly importing from env var in both.